vault/token_store.go

```diff
@@ -290,8 +290,9 @@ func (ts *TokenStore) SaltID(id string) string {
 }
 
 // RootToken is used to generate a new token with root privileges and no parent
-func (ts *TokenStore) rootToken() (*TokenEntry, error) {
+func (ts *TokenStore) rootToken(id string) (*TokenEntry, error) {
```

Comment:
It doesn't look like we need to provide this. Its probably simpler to just parse the output UUID to get the bytes for XOR, instead of changing the interface. Also prevents accidental use later.